samples/excel/85-preview-apis/worksheet-auto-filter.yaml

```diff
@@ -0,0 +1,164 @@
+            await Excel.run(async (context) => {
+                const sheet = context.workbook.worksheets.getActiveWorksheet();
+                sheet.autoFilter.remove();
+                await context.sync();
+            });
+        }
+
+        async function setup() {
+            await Excel.run(async (context) => {
+                const dataSheet = await OfficeHelpers.ExcelUtilities.forceCreateSheet(context.workbook, "Sample");
+
+                const data = [
+                    ["Farm", "Type", "Classification", "Crates Sold Wholesale"],
+                    ["A Farms", "Lime", "Organic", 2000],
+                    ["A Farms", "Lemon", "Organic", 1800],
+                    ["A Farms", "Orange", "Organic", 2200],
+                    ["B Farms", "Lime", "Conventional", 1000],
+                    ["B Farms", "Lemon", "Conventional", 1230],
+                    ["B Farms", "Orange", "Conventional", 800],
+                    ["B Farms", "Orange", "Organic", 500],
+                    ["B Farms", "Lemon", "Organic", 770],
+                    ["B Farms", "Kiwi", "Conventional", 300],
+                    ["B Farms", "Lime", "Organic", 400],
+                    ["C Farms", "Apple", "Organic", 220],
+                    ["C Farms", "Kiwi", "Organic", 120],
+                    ["D Farms", "Apple", "Conventional", 3000],
+                    ["D Farms", "Apple", "Organic", 2800],
+                    ["E Farms", "Lime", "Conventional", 2700],
+                    ["E Farms", "Orange", "Conventional", 2000],
+                    ["E Farms", "Apple", "Conventional", 2200],
+                    ["E Farms", "Kiwi", "Conventional", 1500],
+                    ["F Farms", "Kiwi", "Organic", 150],
+                    ["F Farms", "Lemon", "Conventional", 270]
+                ];
+                const range = dataSheet.getRange("A1:D21");
+                range.values = data;
+                dataSheet.getRange("A1:D1").format.font.bold = true;
+                range.format.autofitColumns();
+                dataSheet.activate();
+                dataSheet.names.add("FarmData", range);
+                await context.sync();
+            });
+        }
+
+        /** Default helper for invoking an action and handling errors. */
+        async function tryCatch(callback) {
+            try {
+                await callback();
+            } catch (error) {
+                // Note: In a production add-in, you'd want to notify the user through your add-in's UI.
+                console.error(error);
+            }
+        }
+    language: typescript
+template:
+    content: |-
+        <section class="ms-font-m">
+          <p>This sample shows how to work with an AutoFilter on a worksheet.</p>
+        </section>
+
+        <section class="samples ms-font-m">
+          <h3>Setup</h3>
+          <button id="setup" class="ms-Button">
+                <span class="ms-Button-label">Create table</span>
```

Comment:
Seems like a misleading button name. It doesn't actually create an Excel.Table object.